src/locales/es/battle.json

```
@@ -32,6 +32,7 @@
   "attackMissed": "¡{{pokemonNameWithAffix}}\nha evitado el ataque!",
   "attackHitsCount": "N.º de golpes: {{count}}.",
   "rewardGain": "¡Has conseguido\n{{modifierName}}!",
+  "rewardGainCount": "¡Has conseguido\n{{count}} {{modifierName}}!",
```

```suggestion
  "rewardGainCount": "¡Has conseguido\n{{count}} $t(modifierName, {\"count\": {{count}} })!",
```

For possible plural handling